src/js/components/TaskDetailComponent.jsx

```diff
@@ -72,16 +65,16 @@ var TaskDetailComponent = React.createClass({
         {<div>
           <dl className="dl-horizontal">
             <dt>Host</dt>
-            <dd>{task.get("host")}</dd>
+            <dd>{task.host}</dd>
             <dt>Ports</dt>
-            <dd>[{task.get("ports").toString()}]</dd>
+            <dd>[{task.ports.toString()}]</dd>
             <dt>Status</dt>
-            <dd>{task.get("status")}</dd>
+            <dd>{task.status}</dd>
             {timeFields}
             <dt>Version</dt>
             <dd>
-              <time dateTime={task.get("version")}>
-                {task.get("version").toLocaleString()}
+              <time dateTime={task.version}>
+                {new Date(task.version).toLocaleString()}
```

Comment:
I don't know how this issue is affected by this, because I can't reproduce this.
I have the correct local time and timezone offset here.
It works like expected.